### PR TITLE
v1.37 mobile changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/mobile-changelog.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+- Release Date: November 16, 2020
+- Server Versions Supported: Server v5.25+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.25 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.19 has ended and upgrading to server ESR v5.25 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Improvements
+ - Icon overhaul.
```
Comment (amyblais) [inline source/administration/mobile-changelog.md:13]: May need more details.

Comment (enahum) [reply source/administration/mobile-changelog.md:13]: @matthewbirtch ☝️

File: source/administration/mobile-changelog.md [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+ - The user autocomplete now allows matching on terms with spaces (For example: @firstname lastName) the same way as in the WebApp.
+
+### Bug Fixes
+
+#### All apps
+ - Fixed an issue where the app crashed when a user received a notification while dismissing the Add Reactions modal.
```
Comment (enahum) [inline source/administration/mobile-changelog.md:19]:
```suggestion
 - Fixed an issue where the app crashed when a user received a notification while dismissing a modal.
```

**Review by enahum — CHANGES_REQUESTED** [review]

File: source/administration/mobile-changelog.md (hunk 1 above)
Comment (matthewbirtch) [reply source/administration/mobile-changelog.md:13]: How about this:
Icons have been updated across the app to be more consistent and adhere to new design standards.

**Review by matthewbirtch — CHANGES_REQUESTED** [review]

**Review by matthewbirtch — APPROVED** [review]

**Review by enahum — APPROVED** [review]

**Review by esethna — APPROVED** [review]

**Pull request merged**